docs/adoption.md

```diff
@@ -97,6 +97,12 @@ Once you have added a remote backend to your Terraform and created a service pri
       | `AZURE_SUBSCRIPTION_ID`  | Navigate to subscriptions and select the Subscription ID for your subscription                                        | YES        |
       | `STATIC_SITE_NAME`       | The name of your static site                                                                                          | YES        |
       | `CUSTOM_DOMAIN_NAME`     | The custom domain name you wish to attach to your static site. See [below](adoption.md#notes-adding-a-custom-domain-optional). | NO         |
+      | `REPAIRS_API_BASE_URL`           | Housing repairs online API URL, this can obtained from the App Service the API was deployed to           | YES     |
+       `REPAIRS_API_IDENTIFIER`         | A unique identifier used to validate access in production                                              | YES |
+     | `REPAIRS_API_BASE_URL_STAGING`           | Housing repairs online API Staging URL, this can obtained from the App Service the API was deployed to           | YES     |
+       `REPAIRS_API_IDENTIFIER_STAGING`         | A unique identifier used to validate access in staging                                              | YES | 
```

Comment:
This line is missing a pipe `|` before the value and the could result in the table not rendering correctly.